executor/insert_test.go

```diff
@@ -15,6 +15,7 @@ package executor_test
 
 import (
 	"fmt"
+	"github.com/pingcap/tidb/types"
```

Comment:
Please format this statement.

Reply:
fixed but the unit test failed without any error.